pkg/apitype/history.go

```diff
@@ -29,6 +29,8 @@ const (
 	PreviewUpdate UpdateKind = "preview"
 	// RefreshUpdate is an update that came from a refresh operation.
 	RefreshUpdate UpdateKind = "refresh"
+	// RenameUpdate is an update that changes the name of a Pulumi program.
```

Comment:
Looks good, but to be super clear, we should probably say "changes the stack name or project name of a Pulumi program." (As "name" could be ambiguous.)